index.js

```diff
@@ -482,11 +482,11 @@
 
             items = _this.sorter(items);
 
-            items = items.slice(0, this.options.items);
+            items = items.slice(0, (_this.options.delimiter === '@') ? this.options.items : this.options.maxGlossaryItems);
```

Comment:
Can you explain why do you need this? Are you limiting the number of results to display after the call to the server? Shouldn't we limit the number of results from the server instead?

Reply:
Correct. Glossary item limit is set by the search.